includes/tuque_wrapper.inc

```diff
@@ -240,7 +240,15 @@ class IslandoraFedoraObject extends FedoraObject {
       return $ret;
     }
     catch (Exception $e) {
-      watchdog('islandora', 'Failed to ingest datastream @datastream on object: @pid</br>code: @code<br/>message: @msg', array(
+      if ($e instanceof DatastreamExistsException) {
+        $message = islandora_deprecated('7.x-1.10', t('The datastream @dsid already exists on the object: @pid. After the 7.x-1.10 release this will throw an exception.', array(
```

Comment:
Just as an example of the nagging, the updated code for [islandora_rest](https://github.com/discoverygarden/islandora_rest/pull/22/files#diff-002a1315b6c97d2d6512f550ca8672adR150) would continue to nag, even though it handles the exception.